### PR TITLE
Development

### PR DESCRIPTION
**Pull request by iraleigh**

### CODE REVIEW

**Comment by iraleigh** [comment]
Everything looks good to me, but on tuesday I would love for a run down on how the piping works in our OS.

File: bin/mutexTest.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+        case 6:
+            OS.mutexLock.acquire(this.var.test3);
+            this.program_counter++;
+            break;
+        case 7:
+            OS.mutexLock.release(this.var.test3);
+            this.program_counter++;
+            break;
+        case 8:
+            OS.mutexLock.release(this.var.test2);
```
Comment (mwishoff) [inline bin/mutexTest.js]: Typo here. We this.var.test should be released. I'll fix it when I get to school today.

File: bin/mutexTest.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+            arrayOfMutexes[0].MutexToString();
+            arrayOfMutexes[1].MutexToString();
+            OS.mutexLock.release(this.var.test3);
+            arrayOfMutexes[0].MutexToString();
+            OS.mutexLock.release(this.var.test);
+
+            arrayOfMutexes[0].MutexToString();
+
+            OS.mutexLock.release(this.var.test2);
```
Comment (mwishoff) [inline bin/mutexTest.js]: Also lines 64-76 need to be deleted or commented out.

**Comment by mwishoff** [comment]
Everything for the most part looks good to me, there are just 2 things I'm going to change in the mutexTest.js file

**Pull request merged**